include/aspect/utilities.h

```diff
@@ -190,6 +190,11 @@ namespace aspect
     bool fexists(const std::string &filename);
 
     /**
+     * Retrieve the last line of a file
+     */
+    std::string get_last_line(std::istream *in);
```

Comment:
I think the argument would be better as a reference, rather than a pointer to a stream.